SubEthaEdit-Mac/Source/NSMenuTCMAdditions.m

```diff
@@ -13,7 +13,7 @@
 
 @implementation  NSMenuItem (NSMenuItemTCMAdditions)
 - (id)autoreleasedCopy {
-    __autoreleasing NSMenuItem *result=[[NSMenuItem alloc] initWithTitle:[self title] action:[self action] keyEquivalent:[self keyEquivalent]];
+    NSMenuItem *result=[[NSMenuItem alloc] initWithTitle:[self title] action:[self action] keyEquivalent:[self keyEquivalent]];
```

Comment:
Arc automatically does the right thing in terms of autrelease when returning values from a method. This is only called autoreleasedCopy for coneniences sake, and because the addition did not want to make NSMenuItem comply with NSCopying. However, this now is slightly awkward. Let us live with that for now.